### PR TITLE
+ORDI, STX, APE markets

### PR DESCRIPTION
**Pull request by gmxer**

### CODE REVIEW

File: config/tokens.ts [hunk 1]
```diff
@@ -55,6 +55,13 @@ const config: {
   [network: string]: TokensConfig;
 } = {
   arbitrum: {
+    APE: {
+      address: "0x74885b4d524d497261259b38900f54e6dbad2210",
+      decimals: 18,
+      transferGasLimit: 200 * 1000,
+      dataStreamFeedId: "0x000316d702a8e25e6b4ef4d449e3413dff067ee77dd366f0550251c07daf05ee",
+      dataStreamFeedDecimals: 18,
```
Comment (xvi10) [inline config/tokens.ts:63]: oracleTimestampAdjustment should be 1

File: config/markets.ts [hunk 2]
```diff
@@ -320,6 +320,58 @@ const config: {
...
+      virtualTokenIdForIndexToken: hashString("PERP:APE/USD"),
+      virtualMarketId: hashString("SPOT:APE/USD"),
+
+      ...baseMarketConfig,
+
+      positionImpactExponentFactor: decimalToFloat(22, 1), // 2.2
+      positivePositionImpactFactor: decimalToFloat(25, 11), // 0.05% for ~90,000 USD of imbalance
+      negativePositionImpactFactor: decimalToFloat(5, 10), // 0.05% for ~45,000 USD of imbalance
+
+      positiveMaxPositionImpactFactor: decimalToFloat(5, 3), // 0.5%
```
Comment (xvi10) [inline config/markets.ts]: should remove this since it is the default value

File: config/markets.ts [hunk 3]
```diff
@@ -320,6 +320,58 @@ const config: {
...
+      openInterestReserveFactorLongs: percentageToFloat("90%"),
+      openInterestReserveFactorShorts: percentageToFloat("90%"),
+
+      maxPnlFactorForAdlLongs: percentageToFloat("80%"),
+      maxPnlFactorForAdlShorts: percentageToFloat("80%"),
+
+      maxLongTokenPoolUsdForDeposit: decimalToFloat(1_500_000), // x1.5 of max open interest
+      maxShortTokenPoolUsdForDeposit: decimalToFloat(1_500_000), // x1.5 of max open interest
+
+      maxLongTokenPoolAmount: expandDecimals(3439653, 18), // ~2M USD (x2 of max open interest)
```
Comment (xvi10) [inline config/markets.ts]: would be good to add underscores to 3439653 for readability

**Review by xvi10 — APPROVED** [review]

File: config/markets.ts [hunk 4]
```diff
@@ -320,6 +320,52 @@ const config: {
...
+      positivePositionImpactFactor: decimalToFloat(25, 11), // 0.05% for ~90,000 USD of imbalance
+      negativePositionImpactFactor: decimalToFloat(5, 10), // 0.05% for ~45,000 USD of imbalance
+
+      positiveSwapImpactFactor: decimalToFloat(15, 9), // 1.5e-8
+      negativeSwapImpactFactor: decimalToFloat(3, 8),
+
+      borrowingFactorForLongs: decimalToFloat(160, 10), // 1.60E-08, ~50% if 100% utilized
+      borrowingFactorForShorts: decimalToFloat(160, 10), // 1.60E-08, ~50% if 100% utilized
+
+      fundingIncreaseFactorPerSecond: decimalToFloat(116, 14), // 0.00000000000116, at least 3.5 hours to reach max funding
```
Comment (gdev8317) [inline config/markets.ts]: Chaos suggested different funding increase factor 1.74E-12, is it expected to have 1.16e-12? 

checking here https://www.notion.so/chaos-labs/New-Asset-Listing-APE-4e70bdbaf75c4ca084eca86024aa7759

File: config/markets.ts [hunk 5]
```diff
@@ -320,6 +320,52 @@ const config: {
...
+      fundingDecreaseFactorPerSecond: decimalToFloat(0), // not applicable if thresholdForDecreaseFunding = 0
+
+      maxFundingFactorPerSecond: decimalToFloat(150, 10), // 0.00000150%,  0.1296% per day, ~47.3% per year
```
Comment (gdev8317) [inline config/markets.ts]: and they recommended 2.25E-08 for max funding factor

**Pull request merged**